setup.bash

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Setup Manubot
+# Based on https://github.com/manubot/rootstock/blob/main/SETUP.md
+# This is designed to be run from the terminal on linux
+
+# Stop on first error.
+set -e
+
+usage() { echo "Usage: $0 [-o <owner-string>] [-r <repo-string>]" 1>&2; exit 1; }
```

Comment:
in general we use long args for readability. Would it be possible to support --owner and --repo (or whatever we'd call the long argument)?

Reply:
Done, changed from using `optargs` to `optarg` which is not built into bash but should come as standard on linux (I believe).

Reply:
Looks like getopt is also on macOS: https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/getopt.3.html